.buildkite/heartbeat/heartbeat-pipeline.yml

```diff
@@ -104,10 +104,19 @@ steps:
           - "heartbeat/build/*.json"
         notify:
           - github_commit_status:
-              context: "Heartbeat: Win-2022 Unit Tests"
+              context: "heartbeat: Win-2022 Unit Tests"
 
-      - label: ":ubuntu: Heartbeat Go Integration Tests"
+      - label: ":ubuntu: Heartbeat Go (Module) Integration Tests"
         command: |
+          set -euo pipefail
+          echo "~~~ Installing @elastic/synthetics"
+          npm install -g @elastic/synthetics
```

Comment:
note for later, raise a ticket to add this to the VM image.

Reply:
Added to META